ansible/configs/osp-migration/infra.yml

```diff
@@ -101,6 +101,10 @@
         openstack_project_services: {}
       when: openstack_project_services == ""
 
+    - debug:
+        msg: "Access to the documentation: {{ document_link }}"
+      when: document_link | default('') != ''
```

Comment:
Didn't you mean to use `agnosticd_user_info` module here ?

Reply:
updated, could you check?